metadata/UiPathDocumentOCR__6__metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "changeLog": "Model: 21.10.1",
+  "cpu": 0,
+  "description": "Machine Learning model for extracting text and checkboxes/radio buttons from Documents. Please see more details including supported languages and link to Activities guide in the Document Understanding Guide here: https://docs.uipath.com/document-understanding/v2021.4/docs/ml-packages#uipath-document-ocr. For alternative deployment methods in high-volume scenarios see also this page: https://docs.uipath.com/document-understanding/v2021.4/docs/ocr-services",
+  "displayName": "UiPathDocumentOCR",
+  "gpu": 0,
+  "inputDescription": "ML Skills deployed using this ML Package can be integrated into RPA workflows using the UiPath Document OCR activity from OCR.Activities pack in the Official feed. File formats accepted include pdf, tiff, jpg or png files. In non-airgapped deployments, the activity requires the Document Understanding API Key input which you need to obtain from your UiPath Automation Cloud account, in the Licenses -> Other Services view.",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON37_DU",
+  "name": "UiPathDocumentOCR",
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "GPU",
+  "projectId": "[project-id]",  
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v21.10",
+  "languageVersion": 3,
+  "baseImageCpu": "python37du:v21.8.1-2",
```

Comment:
From where should I get those?

Reply:
it would be -3 if languageVersion is 3 but that image isn't ready yet
Typically it follows <language>:v21.8.1-<langversion>-<gpu for gpu> but get it confirmed from AIC team too.

Reply:
@vnaren23 Is `-3` image available?